CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
 
 namespace CompanyEmployees.Core.Domain.Entities
```

Comment:
The 'Salary' property should be nullable if it's optional or not always set. Consider using 'decimal?' to align with nullable reference type practices and avoid default 0.0 values that may be misleading.

```suggestion
public decimal? Salary { get; set; }
```